taker-frontend/src/App.tsx

```diff
@@ -110,6 +112,11 @@ export const App = () => {
 
     dayjs.extend(relativeTime);
     dayjs.extend(utc);
+    dayjs.extend(isBetween);
+
+    // Show promo banner from 04.04.2022 until 18.04.2022 (specify midnight the
+    // next day as the end)
+    const isWithinPromoPeriod = dayjs().utc().isBetween("2022-04-04", "2022-04-19");
```

Comment:
I propose to show this sooner. 
```suggestion
    const isWithinPromoPeriod = dayjs().utc().isBetween("2022-03-29", "2022-04-19");
```